build/__phpVersions.sh

```diff
@@ -1,7 +1,7 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-PHP_BUILD_BASE_TAG='20190716.3'
-PHP_RUNTIME_BASE_TAG='20190716.3'
+PHP_BUILD_BASE_TAG='20190710.4'
```

Comment:
Between these versions the `libssl1.0.2` library is missing. I have meanwhile started a base image build but wanted to put this change in to kick off a CI build. Once I test successfully with the latest base build image i will create a new PR.

![image](https://user-images.githubusercontent.com/6559784/61563542-12f5f200-aa29-11e9-810b-5470046d622e.png)